extensions/emmet/src/abbreviationActions.ts

```diff
@@ -184,47 +184,54 @@ export function wrapIndividualLinesWithAbbreviation(args: any) {
 	}
 
 	const editor = vscode.window.activeTextEditor;
-	const selection = editor.selection;
-	let rangeToReplace: vscode.Range = selection.isReversed ? new vscode.Range(selection.active, selection.anchor) : selection;
-	if (!rangeToReplace.isSingleLine && rangeToReplace.end.character === 0) {
-		let previousLine = rangeToReplace.end.line - 1;
-		let lastChar = editor.document.lineAt(previousLine).text.length;
-		rangeToReplace = new vscode.Range(rangeToReplace.start, new vscode.Position(previousLine, lastChar));
-	}
-	if (rangeToReplace.isEmpty) {
-		vscode.window.showInformationMessage('Select more than 1 line and try again.');
-		return;
-	}
+	let rangesToReplace: vscode.Range[] = [];
+	editor.selections.forEach(selection => {
+		let rangeToReplace: vscode.Range = selection.isReversed ? new vscode.Range(selection.active, selection.anchor) : selection;
+		if (!rangeToReplace.isSingleLine && rangeToReplace.end.character === 0) {
+			let previousLine = rangeToReplace.end.line - 1;
+			let lastChar = editor.document.lineAt(previousLine).text.length;
+			rangeToReplace = new vscode.Range(rangeToReplace.start, new vscode.Position(previousLine, lastChar));
+		}
+		if (rangeToReplace.isEmpty) {
```

Comment:
Pull this check out of this foreach loop

`if (editor.selections.find(x => x.isEmpty)) { }`